### PR TITLE
feat: developer concepts 2

### PR DESCRIPTION
**Pull request by MohanNarayana**
Signed-off-by: Mohan Narayana <mohan.narayana@mailchimp.com>

### CODE REVIEW

**Review by Akarshit — APPROVED** [review]

File: guides/developers-guide/concepts/dev-how-do-i.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
+# For Developers: How Do I...?
```
Comment (brent-hoover) [inline guides/developers-guide/concepts/dev-how-do-i.md]: Almost all of this stuff is duplicates from what I already added in the How-to guide. Can you de-duplicate and resubmit?

**Review by brent-hoover — CHANGES_REQUESTED** [review]
This needs to be checked against the How-to guide and remove duplicates. Check against the original doc for what should be where

**Comment by MohanNarayana** [comment]
Doc for context is in [this](https://github.com/reactioncommerce/docs/pull/50#pullrequestreview-859723343) PR

File: guides/developers-guide/concepts/dev-how-do-i.md (hunk 1 above)
Comment (MohanNarayana) [reply guides/developers-guide/concepts/dev-how-do-i.md]: Done

File: guides/developers-guide/concepts/simple-schemas.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Schemas
```
Comment (brent-hoover) [inline guides/developers-guide/concepts/simple-schemas.md:1]: Is this not a duplicate?

File: guides/developers-guide/concepts/understanding-events.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
+# App Events
```
Comment (brent-hoover) [inline guides/developers-guide/concepts/understanding-events.md:1]: This is also a duplicate and also quite a big out of date

**Comment by brent-hoover** [comment]
Closing to be reopened in a later phase

**Pull request closed without merging**